docs/source/appendices/install-with-lxd.md

```diff
@@ -0,0 +1,43 @@
+# Installing bigchainDB on LXC containers using LXD
```

Comment:
This is minor point, but could you change this title to Title Case (and always capitalize BigchainDB; it's a proper noun), i.e.

```text
# Installing BigchainDB on LXC Containers Using LXD
```